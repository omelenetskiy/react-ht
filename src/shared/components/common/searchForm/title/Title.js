import React from 'react';

import StyledTitle from '../../../styled/title';

const Title = ({ children }) => <StyledTitle>{children}</StyledTitle>;

export default Title;
