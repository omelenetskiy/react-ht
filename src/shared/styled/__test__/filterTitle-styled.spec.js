import React from 'react';
import renderer from 'react-test-renderer';
import 'jest-enzyme';
import 'jest-styled-components';

import Title from '../filterTitle';

const theme = {
  stylishColor: 'stylishColor',
};

const props = {
  color: 'light',
};

describe('Title styled', () => {
  it('should render correctly', () => {
    const wrapper = <Title theme={theme} {...props} />;

    const tree = renderer.create(wrapper).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
