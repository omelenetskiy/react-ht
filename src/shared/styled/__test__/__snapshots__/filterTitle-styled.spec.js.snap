// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title styled should render correctly 1`] = `
.c0 {
  color: #fff;
  font-weight: bold;
  font-size: 13px;
}

<span
  className="c0"
  color="light"
/>
`;
